cpp/server/handler.cc

```diff
@@ -3,160 +3,77 @@
 #include <algorithm>
 #include <event2/buffer.h>
```

Comment:
I *think* this one might not be needed either, with `ExtractChain` having moved? Not 100% sure about `event2/http.h`, though.